### PR TITLE
Lab 1

### PR DESCRIPTION
**Pull request by jrlivesey**
@gitK2

### CODE REVIEW

File: Lab_1.ipynb [hunk 1]
```diff
@@ -0,0 +1,432 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4f6414ff-196a-4296-a36c-db21833666e5",
+   "metadata": {},
+   "source": [
+    "As we discussed in lecture, the \"sigma\" value of a given measurement corresponds to the probability of obtaining that measurement from some background distribution. Our first task is to determine the probabilities corresponding to some different \"sigma\" values, to better understand what is meant. Let's try several from $0.25\\sigma$ to $5\\sigma$.\n",
+    "\n",
+    "The standard normal distribution $\\mathcal{N}(0,1)$ is centered on $\\mu=0$ has a standard deviation of $\\sigma=1$. As a result, computing the cumulative distribution function (CDF) at some value $x_\\text{max}$ and subtracting 1 gives the probability that we measure $x \\geq x_\\text{max}$, and this is the probability corresponding to $x_\\text{max}\\sigma$. So if we, for instance, want to determine the probability corresponding to $5\\sigma$, we must compute $1 - \\text{cdf}(x_\\text{max})$ at $x_\\text{max} = 5$."
```
Comment (gitK2) [inline Lab_1.ipynb]:
```suggestion
    "The standard normal distribution $\\mathcal{N}(0,1)$ is centered on $\\mu=0$ has a standard deviation of $\\sigma=1$. As a result, computing the cumulative distribution function (CDF) at some value $x_\\text{max}$ and subtracting from 1 gives the probability that we measure $x \\geq x_\\text{max}$, and this is the probability corresponding to $x_\\text{max}\\sigma$. So if we, for instance, want to determine the probability corresponding to $5\\sigma$, we must compute $1 - \\text{cdf}(x_\\text{max})$ at $x_\\text{max} = 5$."
```

**Comment by gitK2** [comment]
I think you did a great job with this lab. You added more graphs and equations than what I had, and used them well to explain your ideas. 
One suggestion I have is to put titles and/or labels on the axes of your graphs to make them easier to read. Right now I can understand them but it takes some thinking to figure out the X and Y axes correspond to which value.
Another suggestion I have is to put sigmas on your list converting probabilities to sigmas. That would make it consistent with the previous table where the sigmas were all labeled with a sigma symbol.
Finaly, you can discuss what implications discrete probabilities and sigmas have on experiments (non-continuous distributions part 1.D). This is a tricky question that I might not have gotten right either.

**Comment by jrlivesey** [comment]
Thanks! I've made the recommended changes.

**Comment by jrlivesey** [comment]
Submitting this assignment. @hrzhao76

**Pull request merged**